app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -0,0 +1,77 @@
+require 'octokit'
+
+class User
+  class ReputationToken
+    class AwardForPullRequest
+      include Mandate
+
+      initialize_with :action, :github_username, :params
+
+      def call
```

Comment:
This was more what I meant with my comment.

I think having the `merged?` guard inside `award_reputation_to_author` was correct.

```suggestion
      def call
        return unless closed?
        
```